migrator/src/main/java/uk/nhs/digital/ps/migrator/model/nesstar/PublishingPackage.java

```diff
@@ -56,6 +59,23 @@ public String getAbstract() {
             .collect(Collectors.toList());
     }
 
+    public List<String> getKeywords() {
+
+        // Get TaxonomyLookup
+        TaxonomyLookup taxonomyLookup = new TaxonomyLookup();
+
+        // Get the nesstar keywords, and map to hippo taxonomy keys
+        return keywordsXpath.evaluate(rootElement).stream()
+            .flatMap(element -> {
```

Comment:
As discussed we can use `.map()` to get nesstarNameToKeyword and then `.filter()` to strip out the ones that don't exist.